src/inc/til/point.h

```diff
@@ -123,6 +123,38 @@ namespace til // Terminal Implementation Library. Also: "Today I Learned"
             }
         }
 
+        constexpr bool operator<=(const point& other) const noexcept
+        {
+            if (_y < other._y)
+            {
+                return true;
+            }
+            else if (_y > other._y)
+            {

```

Comment:
it's hard to compare points _absoutely_. @miniksa do we need a wrapper that ensures they're within the same bounds? or is this safe?

Reply:
I think sans context, this is a fair comparison to order them as it's the coordinate space and primary ordering across the spaces we generally deal with. (Left to Right, Top to Bottom)

I think we can rely on `Viewport` or eventually consuming and improving it into `til::rectangle` to provide relations in other "walk" directions or within a different coordinate space.